### DISCUSSION TITLE
[Office Hours] Making changes to Task 4

### DISCUSSION THREAD

**Discussion by kc224** [CS342/.github#52] [Office Hours] [answered]
### Related Discussion

N/A

### Reproduction

Attempted to address edits, having some confusion

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
@kc224 As you did not make it to the office hours, can you please provide some more context about your question and the issue you are experiencing? Please edit your existing post with some more information.

**Comment by kc224** [comment, marked as answer]
I'm so sorry about the confusion! I ended up fixing my own problem and thought I had deleted the Office Hours discussion post but didn't see it was still up.

**Reply by PSchmiedmayer** [reply]
Sounds good; thank you for the update! 👍
